filecoindot-nft/src/lib.rs

```diff
@@ -127,7 +130,23 @@ pub mod pallet {
     impl<T: Config> Pallet<T> {
         /// Get the balance of the account
         pub fn balance(who: &T::AccountId) -> u128 {
-            orml_nft::TokensByOwner::<T>::iter_prefix((who,)).count() as u128
+            orml_nft::TokensByOwner::<T>::iter_prefix((who, T::DefaultClassId::get())).count()
```

Comment:
needs a safe cast

Reply:
how come I deleted that, haha. Nice catch